config/CMakeAddFortranSubdirectory.cmake

```diff
@@ -167,9 +180,12 @@ function(cmake_add_fortran_subdirectory subdir)
   set(multiValueArgs LIBRARIES TARGET_NAMES LINK_LIBRARIES DEPENDS CMAKE_COMMAND_LINE)
   cmake_parse_arguments(ARGS "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
   if(NOT ARGS_NO_EXTERNAL_INSTALL)
-    message(FATAL_ERROR
-      "Option NO_EXTERNAL_INSTALL is required (for forward compatibility) "
-      "but was not given." )
+    message("
+-- The external_project ${ARGS_PROJECT} will be installed to to the location
```

Comment:
There is a very minor typo: "... installed to to ..." -> "... installed to ..."

Reply:
I'll fix this.